borg/helpers.py

```diff
@@ -8,6 +8,11 @@
 import os
 import pwd
 import re
+try:
+    from shutil import get_terminal_size
+except ImportError:
+    def get_terminal_size(fallback):
+        return (os.environ.get('COLUMNS', fallback[0]), os.environ.get('LINES', fallback[1]))
```

Comment:
https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size

missing:
- fallback=(80, 24) default
- return value being a named tuple (maybe not needed if we don't use that)